cli/src/semdep/parsers/package_lock.py

```diff
@@ -22,6 +23,96 @@
 logger = getLogger(__name__)
 
 
+def parse_packages_field(deps: Dict[str, JSON]) -> List[FoundDependency]:
+    deps_json = deps.get("packages")
+    if deps_json is None:
+        logger.debug("Found package-lock with no 'packages'")
+        return []
+
+    deps = deps_json.as_dict()
+    try:
+        manifest_deps = set(deps[""].as_dict()["dependencies"].as_dict().keys())
+    except KeyError:
+        manifest_deps = None
+    output = []
+    for package, dep_json in deps.items():
+        fields = dep_json.as_dict()
+        version = fields.get("version")
+        package_name = package[
+            package.rfind("node_modules") + 12 :
```

Comment:
... + 12?

Reply:
The length of the string "node_modules". Basically we want to extract `foo` from something like `node_modules/bar/node_modules/foo`

Reply:
should've been 13 lol